addons/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -61,175 +69,461 @@
  *
  * @author Kai Kreuzer
  * @author Volker Daube
+ * @author Helmut Lehmeyer
+ *         generic DPT Mapper
  * @since 0.3.0
- *
  */
 public class KNXCoreTypeMapper implements KNXTypeMapper {
 
-    static private final Logger logger = LoggerFactory.getLogger(KNXCoreTypeMapper.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(KNXCoreTypeMapper.class);
```

Comment:
Yes, there is no other option. Code guidelines prescribe a non static logger however